clusterloader2/testing/windows-tests/config.yaml

```diff
@@ -0,0 +1,91 @@
+# ASSUMPTIONS:
```

Comment:
I'm not 100% convinced that forking the config is the best option, as it increases maintainance burden, but I can probably live with it.